<launch>

	<node pkg="map_compression" type="renameLaserScans" name="renameLaserScans" output="screen">

	<param name="indexFileName" value="/home/yh/mapModel/2019/07.21/index_kitti.txt"/>
	<param name="isChery" value="false" />	
	<param name="isKITTI" value="true" />	

	<!--param name="loadScanDir" value="/media/yh/YH/DataSet/YQ21/2017_03_09/2017_03_09_drive_0001_sync/velodyne_points/data/"/>
	<param name="saveScanDir" value="/media/yh/YH/its/yq_0901/"/-->
	
	<param name="loadScanDir" value="/media/yh/YH/DataSet/kitti_benchmark_velodyne/dataset/sequences/10/velodyne/"/>
	<param name="saveScanDir" value="/media/yh/YH/its/kitti_10/"/>

	<!--param name="loadScanDir" value="/media/yh/YH/DataSet/chery/scan/"/>
	<param name="saveScanDir" value="/media/yh/YH/its/park/"/-->


  </node>

</launch>
